### ISSUE TITLE
Add active links to BB module usage list

### ISSUE THREAD

**Issue by Norm6257** [beaverbuilder/feature-requests#219]
I'm often faced with updating or cleaning up modules on sites. I do use the filter to display module counts `(is_module_disable_enabled),` but it would be so much more helpful if there was a way to actually determine which posts are containing the module. As it is, it only shows the count of usage, but if there are many posts its extremely difficult and time consuming to determine exactly which posts.

Could the filter be modified to make the counts active links which would then display a listing of the actual posts where the module is used?

 
![image](https://user-images.githubusercontent.com/43974318/150650466-2cbdc3fa-1aac-4d57-b990-6e3f09e4ad45.png)

**Comment by gbissland** [comment]
I agree more information here would be useful, however I could see this sort of query putting a lot of pressure on $2/yr shared hosting servers.

@Norm6257 Try this plugin which I'm sure still works fine. It gives a list of which modules are used on each page https://wordpress.org/plugins/wpbw-beaver-lister/

**Comment by Norm6257** [comment]
I've tried that plugin several times on different sites and it just hangs.  Not sure what is going on with it but for me it's not working.

I don't know the internals of the (is_module_disable_enabled), but it is getting it's usage data from somewhere and it is getting it pretty fast (at least in my experience).  Maybe it already has a list of post IDs that it uses in order to calculate the numbers. If so, I would think it could be relatively quick to build some live links to those posts.  I don't see it being more taxing than just viewing the list of all pages/posts/cpt.
